src/llama.cpp

```diff
@@ -15581,6 +15842,16 @@ struct llm_tokenizer_bpe {
                     "\\p{N}",
                 };
                 break;
+            case LLAMA_VOCAB_PRE_TYPE_CHAMELEON:
+                regex_exprs = {
+                    "<sentinel:[0-9]+>",  // Sentinel tokens
+                    "(IMGIMG)((A|B|C|D|E|F|G|H|I){1,4})Z",  // Image tokens
+                    "([\t\n]|    |  )",  // directly from tokenizer.json
```

Comment:
```suggestion
                    "([\\t\\n]|    |  )",  // directly from tokenizer.json
```

But I think the behavior of the regex would be the same anyway, so feel free to ignore this.